### DISCUSSION TITLE
CWatM initial ground water storage values

### DISCUSSION THREAD

**Discussion by Zprivf** [iiasa/CWatM#87] [Q&A] [answered]
Hello @mikhailsmilovic 

In the ground water module, the initial ground water storage value is assigened from with a comment that it is called from setting file. However, there is no specific map or value called in the setting file (there is a commented out section about initial gwstorage). Is the initial ground water storage assumed value, or is it called from another map, I may have missed a connection here.

Thank you!

**Comment by mikhailsmilovic** [comment, marked as answer]
Guten Tag, we have now updated the code (branch: develop) to allow setting the initial groundwater storage as a value/map [metres] by adding 
storGroundwater = *value* or *path to map* 
in the settings file within the [GROUNDWATER] section.

Thank you for encouraging the update @Zprivf!

Note that this is only used when load_initial = False, and relevant for the linear reservoir representation of groundwater (non-modflow).

Otherwise, in the groundwater module, the initial groundwater storage value is not assigned from the settings file but is assigned from the initialization file (load_initial). If there is no initialization file, the initial value is zero.

**Reply by Zprivf** [reply]
Thank you @mikhailsmilovic 
This is very helpful.
